api-generation/src/main/scala/ai/h2o/sparkling/api/generation/scala/ParametersTemplate.scala

```diff
@@ -59,7 +58,10 @@ object ParametersTemplate extends ScalaEntityTemplate with ParameterResolver {
          |${generateSetters(parameters)}
          |
          |  override private[sparkling] def getH2OAlgorithmParams(): Map[String, Any] = {
-         |    super.getH2OAlgorithmParams() ++
+         |    super.getH2OAlgorithmParams() ++ get${parameterSubstitutionContext.entityName}()
+         |  }
```

Comment:
Sorry, don't see why this code change :(

Reply:
This, is just becuase of Automl. I need to call each parameter group separately in automl (e.g. https://github.com/h2oai/sparkling-water/pull/2120/files#diff-edb3333884f3e1b4c38327d84eab0583R79) and modify it a bit. The original solution could give me all parameters without information to which group the parameter belongs to.

Reply:
Ah, I see! I was having hard time reading it on Github. Makes sense :)